packages/mobile/src/identity/verification.ts

```diff
@@ -666,24 +667,29 @@ export function attestationCodeReceiver(
       action.index
     )
 
-    const attestationInputStatus = yield select(attestationInputStatusSelector)
```

Comment:
This block of code wasn't actually changed, I just moved the `try` further up the flow and added a lock